fireflower/core.py

```diff
@@ -38,6 +39,10 @@ def register_sentry(cls, sentry):
         cls.sentry = sentry
 
     @classmethod
+    def register_datadog_dogstatsd(cls, dogstatsd):
```

Comment:
Add type annotation, for more clarity and less room ambiguity for the reader:
```py
def register_datadog_dogstatsd(cls, dogstatsd: DogStatsD):
```

If you want to avoid the actual reqs.txt dependency, then you can declare the type as a string instead of an actual type:
```py
def register_datadog_dogstatsd(cls, dogstatsd: 'datadog.DogStatsD'):
```